driver/src/main/java/org/neo4j/driver/internal/StandardSession.java

```diff
@@ -70,6 +70,12 @@ public Result run( String statement )
     }
 
     @Override
+    public boolean isOpen()
+    {
+        return connection.isOpen();
```

Comment:
I'm not sure this is correct - wasn't the purpose of `isOpen` to determine if the session object had been closed? If so, this will not work - the connections remain open ideally for the duration of the users application, closing a session object simply returns the connection to the pool to be reused for another session object. 

Ergo; perhaps we can just have a flag in StandardSession that we set on close and check in this method? That'd simplify the code tracking if the underlying connection is open as well.